client/starwhale/base/uricomponents/instance.py

```diff
@@ -0,0 +1,91 @@
+from typing import Any, Dict, Tuple, Optional
+from dataclasses import dataclass
+from urllib.parse import urlparse
+
+from starwhale.utils import config
+from starwhale.base.uri import URI
+from starwhale.base.uricomponents.exceptions import NoMatchException
+
+
+def _get_instances() -> Dict[str, Dict]:
+    return config.load_swcli_config().get("instances", {})
+
+
+def _get_default_instance_alias() -> str:
+    return config.load_swcli_config().get("current_instance", "")
+
+
+def _find_alias_by_url(url: str) -> Tuple[str, str]:
+    """parse url and return instance alias and path from url"""
+    if not url:
+        return _get_default_instance_alias(), ""
+    p = urlparse(url)
+    # use host as alias when url starts with cloud
+    if p.scheme == "cloud":
+        return p.netloc, p.path
+
+    ins_url = "://".join([p.scheme, p.netloc])
+    hits = [name for (name, conf) in _get_instances().items() if conf["uri"] == ins_url]
+    if len(hits) == 1:
+        return hits[0], p.path
+    raise NoMatchException(url, hits)
+
+
+def _check_alias_exists(alias: str) -> None:
+    if alias not in _get_instances().keys():
```

Comment:
`if alias not in _get_instances().keys():` -> `if alias not in _get_instances():` ?